code/modules/jobs/job_types/den.dm

```diff
@@ -43,6 +42,45 @@ Sheriff
 		..()
 		r_pocket = pick(/obj/item/flashlight/flare/torch, /obj/item/flashlight/flare)
 		belt = pick(/obj/item/gun/ballistic/automatic/pistol/m1911, /obj/item/gun/ballistic/revolver/m29, /obj/item/gun/ballistic/revolver/colt6250)
+
+/*
+Deputy
+*/
+/datum/job/den/f13deputy
+	title = "Deputy"
+	flag = F13SHERIFF
+	department_flag = DEN
+	head_announce = list("Security")
+	faction = "Den"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "sheriff"
+	selection_color = "#d7b088"
+	exp_requirements = 30
+	exp_type = EXP_TYPE_DEN
```

Comment:
exp_type is unnecessary, already handled by the parent So is some other stuff, more likely than not.

Reply:
whatever im putting it anyways.

Reply:
/datum/job/den
exp_type = EXP_TYPE_DEN
selection_color = "#d7b088

datum/outfit/job/den
backpack = /obj/item/storage/backpack/satchel/explorer
ears = 			/obj/item/radio/headset/headset_den

now you can just strip it out under every other job because this populates all of /den/ subtype with them
Don't include faction, otherwise people can select job/den and it'll be ugly.